docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,65 @@
+Anyone looking at the PR will be able to quickly see it is not yet ready for review.
+
+Finally, if you are addressing an existing issue, make sure that issue is linked in your PR. If there is not an existing issue, then you should either create an issue or address WHY you are opening the PR specifically.
+If your PR addresses a `Level-5 requirement <https://github.com/orgs/IMAP-Science-Operations-Center/projects/2/views/2>`_, there **must** be a corresponding issue linked.
+
+During review
+=============
+
+As a reviewer, please follow these rules of thumb:
+
+#. Comments should be clear in addressing why you want to see the change
+#. Comments should be polite, but straightforward and candid
+#. If you leave a review, continue to follow up on replies to your questions or comments and review the changes you requested
+#. It is polite, but not required, to provide examples for suggestions (particularly for things like name changes)
+#. If you require a change to be addressed, add a "request changes" comment. If you make one of these comments, it means you are blocking the code review from merging until that change is addressed.
+#. If you make a "request changes" comment, you must create a follow up review where you change that to an approving review. Please do this in a timely matter so you do not block the PR for longer than necessary
+
+As an author:
+
+#. If you would like to request a specific review from someone, make sure they are marked as a reviewer or called out in a comment on the review (by typing ``@<username>``)
+#. You can request a review from the entire team or from a specific instrument team using the team ``IMAP-Science-Operations-Center/imap-sdc``
+
+Before merging
+==============
+
+Before merging, a pull request needs one approving review. While the review is open, anyone can make comments or request changes on the PR.
+
+Although only one approval is required, you must follow these rules:
+
+#. If there is someone with a particular expertise or vested interest in your changes, **do not merge or close the pull request until they get a chance to review.**
+#. Do not merge until you have addressed all the comments on your review, even if you have an approval from someone.
+#. Make sure you have left the review open for a sufficent amount of time to allow people to review - usually 3-5 days is enough.
+#. If someone asked for changes beyond a nitpick, do not merge until you have an approval or thumbs up from them. This does not mean you need to change your code if you don't agree with them, but you should explain why you will not be making the changes and make sure they are ok with merging anyway.
+#. You should go through the :ref:`pull request checklist <pertaining-to-the-code-review>`
+#. You should ensure ALL checks on the PR pass (tests are passing)
```

Comment:
```suggestion
#. You should ensure ALL checks on the PR pass (tests are passing and no linting errors)
```
Do we want to call out linting specifically? Maybe to jargony though...

Reply:
I think it's covered under all checks personally, I mostly added the parentheses as a clarification for what "checks" means. I don't feel strongly about that though, you can commit this suggestion if you think it makes more sense.

Reply:
Nope, this is fine with me!